src/commands/upload_file.ts

```diff
@@ -131,9 +139,9 @@ new CLICommand({
 		ShouldBundleParameter,
 		...ConflictResolutionParams,
 		...DrivePrivacyParameters,
+		CustomContentTypeParameter,
 		LocalFilePathParameter_DEPRECATED,
-		LocalFilesParameter_DEPRECATED,
-		BoostParameter
```

Comment:
Note for the reviewer: 

I found that the `BoostParameter` was duplicated here, it is also above next to `DryRunParameter`